music_assistant/providers/sonos_s1/player.py

```diff
@@ -763,7 +767,7 @@ def _speaker_activity(self, source: str) -> None:
     @soco_error()
     def _join(self, members: list[SonosPlayer]) -> list[SonosPlayer]:
         if self.sync_coordinator:
-            self.unjoin()
+            self._unjoin()
```

Comment:
Think this might be an actual bug its found? Can't test this as I only have one sonos speaker.

Reply:
yep, just checked it